### PR TITLE
Fix flaky ttl_replicated test (remove sleep)

### PR DESCRIPTION
**Pull request by alesapin**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):

### CODE REVIEW

File: tests/queries/0_stateless/00933_ttl_replicated_zookeeper.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
```
Comment (SmitaRKulkarni) [inline tests/queries/0_stateless/00933_ttl_replicated_zookeeper.sh:1]: missed the tags section (-- Tags: replica)

**Review by SmitaRKulkarni — COMMENTED** [review]
Rest all LGTM

**Review by SmitaRKulkarni — APPROVED** [review]

**Comment by alesapin** [comment]
Flaky check -- timeout, it's okay.

**Comment by alesapin** [comment]
`test_dictionaries_redis` -- unrelated.

**Comment by alesapin** [comment]
`Received signal Bus error (7)` -- reported

**Comment by alesapin** [comment]
Stress reported

**Pull request merged**